cmd/create/accountroles/creators.go

```diff
@@ -217,6 +222,71 @@ func getAssumeRolePolicy(file string, input *accountRolesCreationInput) string {
 	})
 }
 
+type hcpManagedPoliciesCreator struct{}
+
+func (hcp *hcpManagedPoliciesCreator) createRoles(r *rosa.Runtime, input *accountRolesCreationInput) error {
+	for file, role := range aws.HCPAccountRoles {
+		accRoleName := aws.GetRoleName(input.prefix, role.Name)
+		if !confirm.Prompt(true, "Create the '%s' role?", accRoleName) {
```

Comment:
I wonder if we're ready to remove this. It's legacy code from when roles were still a new thing in ROSA.

Since the role creation is idempotent, we should probably just confirm `Create the roles? [Y/n]` once and be done with it.

Reply:
or to be honest, if you run `rosa create account-roles`, since it's not a destructive operation we probably shouldn't even confirm.

We don't say `rosa create cluster: are you sure?`. Like, yes, that's what I just typed, thank you. :laughing:

Reply:
Refactored.

Reply:
We do the same for each on operator roles, might be interesting to refactor that when we get there as well

Reply:
I agree, will refactor it when working on `create operator roles` for this epic.